packages/components/stepper/src/js/step-form/step-form.controller.js

```diff
@@ -62,18 +62,34 @@ export default class StepFormController {
     }
   }
 
+  getNodeIndex() {
```

Comment:
What Node Index ? Looks like you try to get the node index of the first step, if this is the case the function should be named `getFirstStepNodeIndex`

Reply:
Node index of the `oui-step-form` that we add in the arrays of steps. In this case, we can keep its DOM position when used with a ngIf or similar.

Reply:
That's not what the name of the function says ;)

Reply:
The name of the function is what it does

Reply:
So I can use it to get the index of any node in the step form ? 😄